index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
Remember to update the content inside your head; see the [HTML best practices](https://mica-web.github.io/learn/html/best-practices#include-a-proper-title-in-your-html-head).